### PR TITLE
Add Layout solution

### PR DESCRIPTION
**Pull request by WaveOfDandelions**
- [DEMO LINK](https://WaveOfDandelions.github.io/layout_catalog/)
- [TEST REPORT LINK](https://WaveOfDandelions.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,7 +6,376 @@
...
+          <a href="#buy" class="card__button">
+            Buy
+          </a>
+        </article>
+
+        <article class="card">
+          <img
+            class="card__product-image"
+            src="./images/imac.jpeg"
+            alt="imac-img"
```
Comment (choeqq) [inline src/index.html:346]: please provide more descriptive alt tag

File: src/index.html [hunk 2]
```diff
@@ -6,7 +6,376 @@
...
```
Comment (choeqq) [inline src/index.html:309]: please provide more descriptive alt tag

File: src/index.html [hunk 3]
```diff
@@ -6,7 +6,376 @@
...
```
Comment (choeqq) [inline src/index.html:272]: please provide more descriptive alt tag

File: src/index.html [hunk 4]
```diff
@@ -6,7 +6,376 @@
...
```
Comment (choeqq) [inline src/index.html:235]: please provide more descriptive alt tag

File: src/index.html [hunk 5]
```diff
@@ -6,7 +6,376 @@
...
```
Comment (choeqq) [inline src/index.html:198]: please provide more descriptive alt tag

**Review by choeqq — CHANGES_REQUESTED** [review]
Please provide DEMO link in PR description

Almost there, all test are passing - just little changes need to be made 🚀

**Review by choeqq — CHANGES_REQUESTED** [review]
Waiting for any updates @WaveOfDandelions 👀

File: src/index.html [hunk 6]
```diff
@@ -6,7 +6,376 @@
...
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <section class="catalog__content">
+        <article class="card" data-qa="card">
```
[the remark this answers is not in the document]
Comment (WaveOfDandelions) [reply src/index.html]: > I don't think `arcitle` is correct tag here, please use plain `div` or `section`

Fixed

File: src/styles/blocks/nav.scss
[the remark this answers is not in the document]
Comment (WaveOfDandelions) [reply src/styles/blocks/nav.scss:1]: > I think you can keep `nav` inside `header` file - header is almost empty without it, and you don't really need to be that granular with BEM blocks

I was change position

File: src/styles/blocks/card.scss [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background: $color-accent-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: all 300ms;
```
[the remark this answers is not in the document]
Comment (WaveOfDandelions) [reply src/styles/blocks/card.scss]: > please list properties you want to use `transition` on, we don't want everything to be transitioned

Now, transition will apply only in hover

File: src/index.html (hunk 1 above)
Comment (WaveOfDandelions) [reply src/index.html]: > please provide more descriptive alt tag

fixed

File: src/index.html (hunk 2 above)
Comment (WaveOfDandelions) [reply src/index.html]: > please provide more descriptive alt tag

fixed

File: src/index.html (hunk 3 above)
Comment (WaveOfDandelions) [reply src/index.html]: > please provide more descriptive alt tag

fixed

File: src/index.html (hunk 4 above)
Comment (WaveOfDandelions) [reply src/index.html]: > please provide more descriptive alt tag

fixed

File: src/index.html (hunk 5 above)
Comment (WaveOfDandelions) [reply src/index.html]: > please provide more descriptive alt tag

fixed

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+    &:hover {
+      cursor: pointer;
+      background-color: $color-accent-white;
+      color: $color-accent-blue;
+    }
+  }
+
+  &:hover {
+    transition: all 300ms;
```
Comment (choeqq) [inline src/styles/blocks/card.scss:80]: `transition` was on correct element - `.card` but transition accepts properties you want to use `transition` on like: 

please move that to `.card` element, and fix `transition` properties
```suggestion
    transition: background-color 300ms, color 300ms, etc...;
```

Comment (WaveOfDandelions) [reply src/styles/blocks/card.scss]: > `transition` was on correct element - `.card` but transition accepts properties you want to use `transition` on like:
> 
> please move that to `.card` element, and fix `transition` properties

fixed

**Review by choeqq — APPROVED** [review]
Nicely done, all tests are passing 🚀

**Pull request closed without merging**